ebos/equil/initstateequil_impl.hh

```diff
@@ -1436,14 +1436,16 @@ InitialStateComputer(MaterialLawManager& materialLawManager,
             }
             else {
                 if (rec[i].gasOilContactDepth() != rec[i].datumDepth()) {
-                    throw std::runtime_error(
```

Comment:
For a three-phase case it seems to be ok, but what happens for a WATER + GAS + VAPWAT case? Does this assume that a gas-oil contact (that is supposed to be ignored in water-gas cases) is read or would that be set as equal to the value from gas-water contact input in Equil keyword?

Reply:
I am not sure if we have proper test cases for the combination of EQUIL + WATER + GAS + VAPWAT where table is not given but contact depth == datum depth.  But based on reading the code I agree that the current implementation seems wrong. It should have been waterOilContactDepth not gasOilContactDepth etc. I can do some testing, but I dont think a potential fix for this should be done in this PR. I would rather open a new one after this is merged.

Reply:
@totto82 I could prepare some test cases and if you wish I can look further into the issue

Reply:
Thanks. That would be great.